app/views/forms/index.html.erb

```diff
@@ -31,7 +31,10 @@
             end
           end
           row.cell(numeric: true) do
-            render FormStatusTagComponent::View.new(status: form.status)
+            html = ""
+            html << render(FormStatusTagComponent::View.new(status: :draft)) if form.has_draft_version
+            html << render(FormStatusTagComponent::View.new(status: :live)) if form.has_live_version
+            html.html_safe
```

Comment:
Why do we need to concat these two renders? 
Cant we just render each option if form is draft/live etc. like we already have here

Reply:
We want both tags to show if a form has been made live and has a draft with edits.

Reply:
Oh, I see now. It might be worth updating the style of the table body block so there isn't a need to concat and then call `html_safe` on that string

```
 <%= table.body do |body| %>
      <% @forms.each do |form| %>
        <% body.row do |row| %>
          <% row.cell do %>
            <% if form.has_live_version %>
              <%= govuk_link_to(form.name, live_form_path(form.id)) %>
            <% else %>
              <%= govuk_link_to(form.name, form_path(form.id)) %>
            <% end %>
          <% end %>
          <%= row.cell(numeric: true) do  %>
            <%= render(FormStatusTagComponent::View.new(status: :draft)) if form.has_draft_version %>
            <%= render(FormStatusTagComponent::View.new(status: :live)) if form.has_live_version %>
          <% end %>
        <% end %>
      <% end %>
    <% end %>
    ```

Reply:
or could abstract it out into a component that given a form, it works out which tags to return 🤷‍♂️

Reply:
Hmm, I'm not sure about the ERB suggested, it is a lot less readable to my eyes. Abstracting it out could work, but I'm not sure I see the necessity, and it would be a bit of work.

Are you happy for us to leave this code as-is for now and come back to it later?